tests/unit/service/conftest.py

```diff
@@ -220,16 +218,27 @@ def online_enable_service_fixture(app_container):
     return app_container.online_enable_service
 
 
+@pytest.fixture(name="mock_get_feature_store_session")
+def mock_get_feature_store_session_fixture():
+    """Mock get_feature_store_session fixture"""
+    with patch(
+        "featurebyte.service.online_enable.SessionManagerService.get_feature_store_session"
+    ) as mock_get_feature_store_session:
+        yield mock_get_feature_store_session
```

Comment:
Revise the test fixtures here to avoid mocking the same reference `featurebyte.service.online_enable.SessionManagerService.get_feature_store_session` in different fixtures.